<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="gaugeBox"/>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="prefButton">
              <property name="text">
               <string>Rider Preferences</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="toDebugButton">
              <property name="text">
               <string>to Debug Console</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="styleSheet">
          <string notr="true">QLabel {font : bold 24px}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="2">
             <widget class="QLCDNumber" name="lcdNumber_7"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>RMS Current</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLCDNumber" name="lcdNumber_12"/>
            </item>
            <item row="1" column="3">
             <widget class="QLCDNumber" name="lcdNumber_4"/>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>DC Voltage</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>RPM</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLCDNumber" name="lcdNumber_5"/>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Motor Temp</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLCDNumber" name="lcdNumber_10"/>
            </item>
            <item row="3" column="1">
             <widget class="QLCDNumber" name="lcdNumber_6"/>
            </item>
            <item row="1" column="0">
             <widget class="QLCDNumber" name="lcdNumber"/>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>drive6stat</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>IQ Current</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLCDNumber" name="lcdNumber_2"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>EMCY6</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Voltage Angle</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>drive7stat</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>EMCY7</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Battery Temp</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLCDNumber" name="lcdNumber_9"/>
            </item>
            <item row="3" column="3">
             <widget class="QLCDNumber" name="lcdNumber_8"/>
            </item>
            <item row="1" column="2">
             <widget class="QLCDNumber" name="lcdNumber_3"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>HS Temp</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLCDNumber" name="lcdNumber_11"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="toDashButton">
              <property name="text">
               <string>to Dashboard</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="styleSheet">
               <string notr="true">QLabel {font : 24px }</string>
              </property>
              <property name="text">
               <string>Regenerative Braking</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="styleSheet">
               <string notr="true">QCheckBox::indicator {
     width: 60px;
     height: 60px;
 }

QCheckBox {
	 font-size: 24px;
}


</string>
              </property>
              <property name="text">
               <string>Exponential Throttle Feel Mapping</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>30</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="cancelButton">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="acceptButton">
              <property name="text">
               <string>Accept Changes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
